website/src/components/index-page.tsx

```diff
@@ -1,24 +1,22 @@
 import { ReactElement } from 'react';
-import dynamic from 'next/dynamic';
 import { HeroGradient, HeroIllustration, NPMBadge } from '@theguild/components';
 import gqlCodegenCover from '../../public/assets/illustrations/gql-codegen-cover.svg';
 import gqlGenerateCodeIllustration from '../../public/assets/illustrations/gql-generate-code-illustration.svg';
 import gqlWatchForChangesIllustration from '../../public/assets/illustrations/gql-watch-for-changes-illustration.svg';
 
-const LiveDemo = dynamic(() => import('@/components/live-demo/LiveDemo'), { ssr: false });
-
 export function IndexPage(): ReactElement {
   return (
     <>
       <HeroGradient
-        title="Generate code from your GraphQL schema"
-        description="Generate code from your GraphQL schema and operations with a simple CLI"
-        link={{
-          href: '/docs/getting-started/installation',
-          children: 'Try It Now',
-          title: 'Get started with GraphQL Code Generator',
-        }}
-        version={<NPMBadge name="@graphql-codegen/cli" />}
+        title="Generate Type-Safe GraphQL Client and Server Code"
+        description="Supercharge Your GraphQL Development Flow with Fully Typed Code in Seconds."
+        link={[
+          {
+            href: '/docs/getting-started/introduction',
```

Comment:
I'm not sure if it's just the preview branch but clicking on the `Get Started with Client and Server` button on the home page takes me to `https://<hash>.graphql-code-generator.pages.dev/docs/getting-started/introduction` which is a 404

`https://<hash>.graphql-code-generator.pages.dev/docs/getting-started` seems to be the correct URL

Reply:
yeah error here, fixed it!